packages/arb-validator/rollupvalidator/rpcserver.go

```diff
@@ -18,52 +18,21 @@ package rollupvalidator
 
 import (
 	"context"
-	"log"
 	"net/http"
 	"time"
 
 	"github.com/offchainlabs/arbitrum/packages/arb-validator-core/validatorserver"
 	"github.com/offchainlabs/arbitrum/packages/arb-validator/rollupmanager"
-
-	"github.com/gorilla/handlers"
-	"github.com/gorilla/mux"
-	"github.com/gorilla/rpc"
-	"github.com/gorilla/rpc/json"
 )
 
 // Server provides an interface for interacting with a a running coordinator
 type RPCServer struct {
 	*Server
 }
 
-func LaunchRPC(man *rollupmanager.Manager, port string) error {
-	server, err := NewRPCServer(man, time.Second*60)
-	if err != nil {
-		return err
-	}
-
-	// Run server
-	s := rpc.NewServer()
-	s.RegisterCodec(json.NewCodec(), "application/json")
-	s.RegisterCodec(json.NewCodec(), "application/json;charset=UTF-8")
-
-	if err := s.RegisterService(server, "Validator"); err != nil {
-		log.Fatal(err)
-	}
-	r := mux.NewRouter()
-	r.Handle("/", s).Methods("GET", "POST", "OPTIONS")
-
-	headersOk := handlers.AllowedHeaders([]string{"X-Requested-With", "Content-Type", "Authorization"})
-	originsOk := handlers.AllowedOrigins([]string{"*"})
-	methodsOk := handlers.AllowedMethods([]string{"GET", "HEAD", "POST", "PUT", "OPTIONS"})
-
-	return http.ListenAndServe(":"+port, handlers.CORS(headersOk, originsOk, methodsOk)(r))
-}
-
 // NewServer returns a new instance of the Server class
-func NewRPCServer(man *rollupmanager.Manager, maxCallTime time.Duration) (*RPCServer, error) {
-	server, err := NewServer(man, maxCallTime)
-	return &RPCServer{server}, err
+func NewRPCServer(man *rollupmanager.Manager, maxCallTime time.Duration) *RPCServer {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [line is 85 characters, out of limit 80](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)